### PR TITLE
[AUD-972] Fill out explainer and tiers of audio page

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Add additional UI to /audio page. Does not connect the challenges yet.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

https://user-images.githubusercontent.com/2731362/156215061-8fb5582a-171f-45b6-8c17-410cc968d1c2.mp4

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

Mobile native release

### CODE REVIEW

File: packages/mobile/src/components/mobile-upload-drawer/MobileUploadDrawer.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const useStyles = makeStyles(({ typography, spacing }) => ({
     width: 80
   },
   title: {
-    fontSize: typography.fontSize.xxxl,
+    fontSize: typography.fontSize.xxxxl,
```
Comment (raymondjacobson) [inline packages/mobile/src/components/mobile-upload-drawer/MobileUploadDrawer.tsx:35]: the web client does 4xl for this. I kinda like that more, but not sure if this was intentional

Comment (sliptype) [reply packages/mobile/src/components/mobile-upload-drawer/MobileUploadDrawer.tsx:35]: I'm on board with `4xl`. Thoughts @dylanjeffers ?

File: packages/mobile/src/screens/audio-screen/Tier.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    width: '100%',
+    justifyContent: 'center',
+    alignItems: 'center',
+    paddingVertical: spacing(10),
+    paddingHorizontal: spacing(4),
+    borderColor: palette.neutralLight7,
+    borderWidth: 2,
+    borderRadius: spacing(4),
+    backgroundColor: palette.neutralLight10,
+    flexDirection: 'column'
```
Comment (sliptype) [inline packages/mobile/src/screens/audio-screen/Tier.tsx:33]: Column is the default flex direction in RN

File: packages/mobile/src/components/mobile-upload-drawer/MobileUploadDrawer.tsx (hunk 1 above)
Comment (sliptype) [reply packages/mobile/src/components/mobile-upload-drawer/MobileUploadDrawer.tsx:35]: I wonder if it makes sense to have a function like `spacing(2)` for font size too

**Review by sliptype — APPROVED** [review]
Looks great!!

File: packages/mobile/src/screens/audio-screen/Tier.tsx (hunk 2 above)
Comment (raymondjacobson) [reply packages/mobile/src/screens/audio-screen/Tier.tsx:33]: ty

Comment (raymondjacobson) [reply packages/mobile/src/screens/audio-screen/Tier.tsx:33]: well we have the h1,h2,h3... styles, which over time, we should figure out just how to keep 1-1 w/ julian's design so we're never off cannon

File: packages/mobile/src/components/mobile-upload-drawer/MobileUploadDrawer.tsx (hunk 1 above)
Comment (dylanjeffers) [reply packages/mobile/src/components/mobile-upload-drawer/MobileUploadDrawer.tsx:35]: i would have loved that, however... numbers as the first value in js :/

File: packages/mobile/src/screens/audio-screen/Tier.tsx (hunk 2 above)
Comment (dylanjeffers) [reply packages/mobile/src/screens/audio-screen/Tier.tsx:33]: yeah totally, and just a headsup im totally guessing on the existing values, so we should all def work to define them more rigorously

**Pull request merged**